services/application.js

```diff
@@ -303,27 +303,28 @@ class Application {
         }
     }
 
+    async remindApplicationSubmission() {
+        const inactiveDuration = this.emailParams.inactiveDays - this.emailParams.remindDay;
+        const remindCondition = {
+            updatedAt: {
+                $lt: subtractDaysFromNow(inactiveDuration),
```

Comment:
This was not working for me, the query was always returning 0 applications. I think the issue is that subtractDaysFromNow is returning a date object. I was able to get it to work by converting to an ISO string before using it in the pipeline here.  ".toISOString()"

Reply:
updated it to 30days.